srcalc/src/test/java/gov/va/med/srcalc/domain/model/AbstractVariableTest.java

```diff
@@ -10,19 +10,19 @@
     /**
      * A 40-character string.
      */
-    private static final String FORTY_CHARS =
+    public static final String FORTY_CHARS =
             "0123456789012345678901234567890123456789";
 
     /**
      * A 41-character string.
      */
-    private static final String FORTY_ONE_CHARS =
+    public static final String FORTY_ONE_CHARS =
             "0123456789012345678901234567890123456789X";
 
     /**
      * An 81-characeter string.
      */
-    private static final String EIGHTY_ONE_CHARS =
+    public static final String EIGHTY_ONE_CHARS =
```

Comment:
Right now this is the only constant that is being used elsewhere. The other two don't necessarily need to be public.